harness/determined/cli/experiment.py

```diff
@@ -796,6 +798,26 @@ def set_gc_policy(args: Namespace) -> None:
         print("Aborting operations.")
 
 
+def get_fmted_checkpoints(unfmted_checkpoints):
```

Comment:
why not sure this out in lines 759-770. i see no need for this intermediary format that is only used to map immediately into another.